README.md

```diff
@@ -1,208 +1,262 @@
+### Step 6 - Include Push Notification Snippet in the layouts
 
-To send a message without using the devices page, simply create a microflow that retrieves the device from a user account and create a message object with the attributes filled. Then simply pass this message to one of the microflows in the use me folder that sends either a message or a list of messages.  
+Include a push notification snippet on mobile and tablet layouts. You can do this by drag-and-dropping the `PushNotification_Snippet` snippet (located in `_USE ME/Snippets` folder in PushNotifications module) to your layout.
 
-## Setting up Apple Push Notification Server
-In order to send push notifications for apple devices from this module you will need to correctly set up and aquire a certificate from apple, then add this to the Mendix settings pages.
+<img src="assets/images/implementation guide/Include push notification snippet on layouts.JPG"/>
 
-### Step 1 - Login to members center
-Login to the [members center] (https://developer.apple.com/) on developer.apple.com. If you do not have an apple developers license you will need to purchase this from apple. When logged in to members center click on "Certificates, Identifiers & Profiles". 
+### Step 7 - Set up the administration pages
 
-This will take you to a screen like this:
-<img src="assets/images/apn-step1.PNG"/>
+Add `AppleAdministration`, `GoogleAdministration`, and `Device_Overview` pages to the project navigation. The `Apple Administration` and `GoogleAdministration` pages are used to configure your application to be able to reach the respective services (APNs and GCM) later on. The `Device_Overview` page is useful for testing purpose.
 
-### Step 2 - Create APP ID
+> Note: don't forget to set the `Project security` -> `User roles` to include `PushNotifications.Administrator` role as part as the main `Administrator` role and `PushNotifications.User` role as part of the main `User` role.
+>
+> <img src="assets/images/implementation guide/Project security.JPG"/>
 
-Click on Identifiers and then click on App IDs.
-On the top right there will be a plus button, press this and a dialog like this should appear:
-<img src="assets/images/apn-step2.PNG"/>
+At this moment you can deploy your application to the cloud. If you are using Free App, simply click the `Run` button.
 
-Enter your app ID name and select Explicit App ID. You must select Explicit App ID inorder to be able to perform push notifications.
-Enter a Bundle ID, this bundle ID must match the bundle ID that you entered during the phonegap build phase.
-<img src="assets/images/apn-step2-1.PNG"/>
-Tick the options push notifications and then click continue.
+### Step 8 - Set up access to APNs and GCM
 
-### Step 3 - Confirm
-Click submit
-<img src="assets/images/apn-step3.PNG"/>
+Set up access to APNs and GCM and configure them in your application.
```

Comment:
'and configure them in your application, using the `AppleAdministration` and `GoogleAdministration` pages respectively.' 

(So people actually know where to go for this :)